File: app/src/main/java/com/vkochenkov/weatherfromopenapis/entities/cities/City.java
package com.vkochenkov.weatherfromopenapis.entities.cities;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class City {
    String name;
    String latitude;
    String longitude;

    public City(String name, String latitude, String longitude) {
        this.name = name;
        this.latitude = latitude;
        this. longitude = longitude;
    }
}

File: app/src/main/java/com/vkochenkov/weatherfromopenapis/data/db/entities/City.kt
package com.vkochenkov.weatherfromopenapis.data.db.entities

import androidx.room.ColumnInfo
import androidx.room.Embedded
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "Cities")
data class City(
    @PrimaryKey val name: String,
    @ColumnInfo(name = "latitude") val latitude: String,
    @ColumnInfo(name = "longitude") val longitude: String
) 